.github/workflows/windows.yml

```diff
@@ -1,7 +1,7 @@
 name: Windows
 
 on:
-  pull_request:
```

Comment:
Was this just temporary for debugging?

Reply:
Yes. I'll remove it before marking this PR as ready.